build-system/runner/src/org/ampproject/AmpCommandLineRunner.java

```diff
@@ -52,7 +52,7 @@
   ImmutableMap<String, Set<String>> suffixTypes = ImmutableMap.of(
       "module$src$log.dev", ImmutableSet.of(
           "assert", "fine", "assertElement", "assertString",
-          "assertNumber", "assertBoolean"),
+          "assertNumber", "assertBoolean", "assertArray"),
```

Comment:
you're going to need to recompile the runner.jar file to get this to work. LMK if you need help (or i can just compile it for you too)

Reply:
Do we have instructions on how to do that anywhere?